include/quiche.h

```diff
@@ -257,6 +257,16 @@ ssize_t quiche_conn_recv(quiche_conn *conn, uint8_t *buf, size_t buf_len);
 // Writes a single QUIC packet to be sent to the peer.
 ssize_t quiche_conn_send(quiche_conn *conn, uint8_t *out, size_t out_len);
 
+typedef struct {
+    // Time to send the packet out.
+    struct timespec send_time;
+} quiche_send_info;
```

Comment:
Looks like `typedef struct SendInfo quiche_send_info` is good enough

Reply:
hmm, ok, but send_time is of different type.

Reply:
Without defining struct quiche_send_info, nginx build with quiche fails, because of unknown type.
`error: field ‘packet_send_info’ has incomplete type`. So this definition is needed here.

Reply:
Oh I see. I thought it's an opaque. LGTM